content/react/pt/get-started.md

```diff
@@ -6,20 +6,20 @@ description: "Configuração do React Storybook no ambiente de desenvolvimento"
 
 # Introdução
 
-Storybook funciona em paralelo á aplicação em modo de desenvolvimento.
-Ajuda na construção de componentes de interface de utilizador isolados de qualquer lógica e contexto da aplicação.
-Esta edição de Aprendizagem de Storybook é destinada para React.
-Encontram-se disponíveis outras edições quer para [Vue](/vue/pt/get-started), quer para [Angular](/angular/pt/get-started).
+Storybook funciona em paralelo à aplicação em modo de desenvolvimento.
+Ajuda na construção de componentes de interface de usuário, isolados de qualquer lógica e contexto da aplicação.
+Esta edição de Aprenda Storybook é destinada para React.
+Encontram-se disponíveis outras edições tanto para [Vue](/vue/pt/get-started), quanto para [Angular](/angular/pt/get-started).
 
 ![Storybook e a aplicação](/storybook-relationship.jpg)
 
-## Configuração de Storybook com React
+## Configurando o Storybook com React
 
-Irão ser necessárias algumas etapas adicionais de forma a ser possível configurar o processo de compilação no nosso ambiente de desenvolvimento.
-Para começar queremos usar o pacote [Create React App](https://github.com/facebook/create-react-app) ou como é vulgarmente conhecido (CRA), para configurar o nosso ambiente local e ativar o modo de testes com [Storybook](https://storybook.js.org/) e 
-[Jest](https://facebook.github.io/jest/) na nossa aplicação.
+Serão necessárias algumas etapas adicionais para que seja possível configurar o processo de compilação em nosso ambiente de desenvolvimento.
```

Comment:
I agree with almost of the change in here, but replacing "no" with "em", it's unnecessary, this will   transform it into a specified version. And not what's intended, it's portuguese in all it's form. Not a specification to pt-BR